### ISSUE TITLE
How to rename query from collection?

### ISSUE THREAD

**Issue by kitze** [redound/graphql-ide#14]
First of all, thank you for this project, it's a game changer :) 

I downloaded the latest version but I cannot seem to find the option to name/rename when saving a query into collection.

Is it a bug or I just cannot find it?

Thanks!

**Comment by orefalo** [comment]
Same issue here... REALLY ANNOYING ;-)

**Comment by olivierandriessen** [comment]
@kitze @orefalo Totally agree this is annoying. Although it's possible to rename query by renaming them inside the editor, this functionality will be improved in the next release. Thanks for pointing out!

**Comment by orefalo** [comment]
?! Inside an editor.. really? I must have missed something, could you provide instructions.

**Comment by olivierandriessen** [comment]
<img width="186" alt="screen shot 2017-02-06 at 20 43 48" src="https://cloud.githubusercontent.com/assets/3968358/22663477/ffe06808-ecac-11e6-889f-2e19cbd8d2bd.png">
You can change it inside the query editor itself. It will automatically save.

**Comment by orefalo** [comment]
![graphql_ide](https://cloud.githubusercontent.com/assets/52085/22665515/02174814-ec83-11e6-9643-362ef5a4bfb8.gif)
 But the field is not editable, see above

**Comment by olivierandriessen** [comment]
@orefalo That's right. Had to be disabled in the initial release the way it's designed. That's it for now.

**Comment by mzohaibqc** [comment]
@kitze 
This is designed this way as you are expecting. Initially I was trying to do that same and also I created a similar issue #23 but later I found how to use this.

Since there is no docs or introductory video for this software / package that's why it's make it difficult to use. However, after hit and trial, I came to know how to set name of a query or mutation or rename it. It is simply name of query itself. If you want to rename it, change the query name.
e.g.
```
query DoIt {
    getFoo(id: 123) {
       name
   }
}
```
So `Doit` is the query name which will be shown in Query title as well as window / tab name.
Here is a screenshot for more clarification.

![screen shot 2017-03-24 at 11 28 33 am](https://cloud.githubusercontent.com/assets/13637803/24283249/6db150ac-1086-11e7-9221-6f9ab806e31f.png)

I hope, it's clear now. Stay happy :)

**Comment by kitze** [comment]
Ah, thank you very much @mzohaibqc, that will work for me but I will leave this issue open so it's clearer for other devs.

Cheers 🙌

**Comment by gunsluo** [comment]
nice！ but, the user not easy to find this feature.
